assets/locales/service.en.toml

```diff
@@ -199,3 +199,12 @@ one = "Cancelled"
 [ReleasePageType]
 description = "Release"
 one = "Release"
+
+[NoReleasesFound]
+description = "No releases found"
+one = "No releases found"
+
+[NoReleasesFoundDescription]
+description = "No Releases Found Description"
+one = "There are no cancelled releases. Try selecting a different type of release."
```

Comment:
"There are no **_cancelled_** releases..."

Variations for published and upcoming releases would also be required to avoid confusing the user.

Reply:
this issues is now resolved.